content/nginx-one/changelog.md

```diff
@@ -32,6 +32,10 @@ h2 {
 
 Stay up-to-date with what's new and improved in the F5 NGINX One Console.
 
+## January 16, 2025
+
+In the Configuration Editor you can now highlight a portion of the configuration and click "Explain with AI" to have the NGINX One AI Assistant give an explanation and guidance. AI Assistant is trained on the official NGINX documentation.
```

Comment:
We haven't mentioned the `Configuration Editor` in our docs. And I'm making an interpretation based on what I've done to "Explain with AI":

```suggestion
In the F5 NGINX One Console, you can now select lines from your configuration files, and then select **Explain with AI**. The NGINX One AI Assistant explains those lines based on the official NGINX documentation.
```